_posts/en/posts/schnorr-sigagg.md

```diff
@@ -0,0 +1,84 @@
+---
+type: posts
+layout: post
+lang: en
+name: schnorr-sigagg
+id: en-schnorr-sigagg
+title: Technology roadmap: Schnorr signatures & signature aggregation
+permalink: /en/2017/03/22/schnorr-sigagg/
+version: 1
+excerpt: Status and explanation of Schnorr signatures and signature aggregation
+---
+
+## Schnorr Signatures
+
+The replacement of Bitcoin’s digital signature algorithm (ECSDA) for the more efficient Schnorr algorithm has long been at the top of the bucket list for many Bitcoin developers. A simple algorithm leveraging elliptic curves cryptography, Schnorr enables several improvements over the existing scheme all while preserving all of the features and security assumptions of ECSDA.
+
+Notably, Schnorr signatures support “native multisig” which enables the aggregation of multiple signatures into a single one valid for the sum of the keys of their respective inputs. This functionality offers three important benefits: 
+
+Constant-size signatures irrespective of the number of participants in the multisig setup. A 50-of-50 transaction is effectively the same size as one that uses a single public key and signature. For this reason, the performance of such schemes is significantly improved by removing the original requirement of validating every signature individually. Additionally, the verification of Schnorr signatures is slightly faster than that of ECSDA.
```

Comment:
Given the previous paragraph's introduction of three benefits, I suggest making this an ordered list.  E.g.

1. Constant-size signatures...

2. The diminished size...

3. From a privacy standpoint